.github/workflows/vulnerability-scanning-on-repo.yml

```diff
@@ -100,48 +106,48 @@ jobs:
       - generate_bom
     runs-on: ubuntu-latest
     steps:
-      - name: Download artifact       
+      - name: Download artifact
         uses: actions/download-artifact@v4
         with:
-          name: ${{ github.run_number }}-sbom.json
+          name: ${{ github.run_number }}${{ inputs.package-directory }}.sbom.json
```

Comment:
From the default "." this would result in ${{ github.run_number }}..sbom.json 
Is that what you want or are you missing a slash or something?

Reply:
Not want I meant to do. Clean up ticket created
https://telicent.atlassian.net/browse/TELFE-518